patches/tModLoader/Terraria/ModLoader/EquipType.cs

```diff
@@ -18,6 +18,7 @@ public enum EquipType
 		Shield,
 		Neck,
 		Face,
+		Beard,
 		Balloon
```

Comment:
So that people don't need to rebuild their balloon enthusiastic mods
```suggestion
		Balloon,
 		Beard,
```